apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientImpl.cs

```diff
@@ -73,14 +73,8 @@ internal SubscriberClientImpl(SubscriptionName subscriptionName, IEnumerable<Sub
         GaxPreconditions.CheckNotNull(settings, nameof(settings));
         settings.Validate();
         // These values are validated in Settings.Validate() above, so no need to re-validate here.
-        _ackExtensionWindow = settings.AckExtensionWindow ?? DefaultAckExtensionWindow;
-        _ackDeadlineSeconds = (int) ((settings.AckDeadline ?? DefaultAckDeadline).TotalSeconds);
-        // For exactly-once delivery subscriptions, AckDeadline value should be the default value of 60 seconds or the user provided value.
-        _ackDeadlineSecondsForExactlyOnceDelivery = (int) ((settings.AckDeadline ?? DefaultAckDeadlineForExactlyOnceDelivery).TotalSeconds);
-        // Ensure the duration between lease extensions is at least MinimumLeaseExtensionDelay (5 seconds).
-        // The minimum allowable lease duration is 10 seconds, so this will always be reasonable.
-        _autoExtendDelay = GetAutoExtendDelay(_ackDeadlineSeconds);
-        _autoExtendDelayForExactlyOnceDelivery = GetAutoExtendDelay(_ackDeadlineSecondsForExactlyOnceDelivery);
+        _normalLeaseTiming = new LeaseTiming(settings.AckDeadline ?? DefaultAckDeadlineForExactlyOnceDelivery, settings.AckExtensionWindow ?? DefaultAckExtensionWindow);
```

Comment:
Typo - Just `DefaultAckDeadline` here.

Reply:
Gah, thank you. (Slightly worrying that this wasn't caught by tests, but hey...)

Reply:
Fixed, will merge when the new commit has gone green.